core/app/assets/styles/utilities/_borders.scss

```diff
@@ -0,0 +1,74 @@
+// stylelint-disable property-blacklist, declaration-no-important
```

Comment:
I don't think we suppose to use all these rules. It's should be consumed as a variable wherever it's needed